people.json

```
@@ -148,6 +147,14 @@
             [
                "Instagram",
                "https://instagram.com/groverwhim"
+            ],
+			[
```

And one more 😂  I'm sorry, I can't stop myself
```suggestion
		[
```